src/applications/mhv/medical-records/components/MrBreadcrumbs.jsx

```diff
@@ -1,27 +1,24 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import { VaBreadcrumbs } from '@department-of-veterans-affairs/web-components/react-bindings';
-// import { replaceWithStagingDomain } from '~/platform/utilities/environment/stagingDomains';
+import { Link } from 'react-router-dom';
 
 const MrBreadcrumbs = () => {
   const crumbs = useSelector(state => state.mr.breadcrumbs.list);
-  const currentPath = useSelector(state => state.mr.breadcrumbs.location);
 
   return (
     <>
       {crumbs.length > 0 && crumbs[0]?.url ? (
         <div
-          className="vads-l-row breadcrumbs-container no-print"
+          className="vads-l-row vads-u-padding-y--3 breadcrumbs-container no-print"
+          label="Breadcrumb"
           data-testid="breadcrumbs"
         >
-          <VaBreadcrumbs label="Breadcrumb" mobileFirstProp>
-            {crumbs.map((crumb, idx) => (
-              <a href={crumb.url} key={idx}>
-                Back to {crumb.label}
-              </a>
-            ))}
-            <a href={currentPath?.url}>{currentPath?.label}</a>
-          </VaBreadcrumbs>

```

Comment:
Was this a specific bug you encountered with the `va-breadcrumbs` design system  component? If so, would you please file a bug report?  We'd love to address it.

Reply:
@micahchiang, no bug -- our UCD team simply had a different design in mind for the MR breadcrumbs that did not require the component.